services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/DataPlaneClient.java

```diff
@@ -26,15 +26,7 @@
 
 @Component
 public interface DataPlaneClient {
-    Map<String, List<GoalStateOperationStatus>> createGoalStates(GoalState goalState, String hostIp) throws Exception;
-
-    List<Map<String, List<GoalStateOperationStatus>>> createGoalStates(List<UnicastGoalState> unicastGoalStates) throws Exception;
-
-    List<Map<String, List<GoalStateOperationStatus>>> updateGoalStates(List<UnicastGoalState> unicastGoalStates) throws Exception;
-
-    List<Map<String, List<GoalStateOperationStatus>>> deleteGoalStates(List<UnicastGoalState> unicastGoalStates) throws Exception;
```

Comment:
A quick question: why do we remove these interfaces?

Reply:
Actually it is okay to keep the interfaces which reminds us that we need to implement in the future.

Reply:
I was thinking about adding it when I needed it.